static/src/stylesheets/head.content.scss

```diff
@@ -43,3 +43,4 @@
 @import 'module/atoms/_interactive';
 @import 'module/atoms/_youtube';
 @import 'module/atoms/_media';
+@import 'module/atoms/_storyquiz';
```

Comment:
Do we really have to inline these? Since there are at the bottom of the article it would be ok to have the CSS in the non-inlined version. What do you think?